app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+
+    people = {}
+
+    def __init__(self, name: str, age: int):
+        self.name = name
+        self.age = age
+        self.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+
+    people_list = []
+
+    for each in people:
```

Comment:
Could you rename `each` variable? This name doesn't tell anything about what stored in it. I'd prefer `person_dict`

Reply:
Thanks for the advice. Fix